src/modules/SX127x/SX127x.cpp

```diff
@@ -836,6 +836,44 @@ int16_t SX127x::disableAddressFiltering() {
   return(_mod->SPIsetRegValue(SX127X_REG_BROADCAST_ADRS, 0x00));
 }
 
+int16_t SX127x::setOokThresholdType(uint8_t type) {
+  // check active modem
+  if(getActiveModem() != SX127X_FSK_OOK) {
+    return(ERR_WRONG_MODEM);
+  }
+
+  int16_t state = ERR_NONE;
+  state = _mod->SPIsetRegValue(SX127X_REG_OOK_PEAK, type, 4, 3, 5);
+
+  return(state);
+}
+
+int16_t SX127x::setOokFixedOrFloorThreshold(short int value) {
```

Comment:
Please use stdint.h types (e.g. uint8_t instead of short int).

Reply:
ok